bin/commands/folder/get.js

```diff
@@ -0,0 +1,24 @@
+const helpers = require('../../helpers')
+
+module.exports = {
+  command: 'get <folder-id>',
+  desc: 'Fetch a single folder.',
+  builder: {},
+  handler: async function (argv) {
+    try {
+      const client = helpers.getClient(argv)
+      const result = await client.getFolder(argv.folderId)
+      if (argv.json) {
+        helpers.printJson(result)
+      } else {
+        helpers.print({
+          message: result.name,
+          id: result._id,
+        })
+      }
+    } catch (error) {
+      throw error
```

Comment:
What's the point of the `try`/`catch` wrapper on these if it just throws? Placeholder for handling it different in the future?

Reply:
Ah good catch, I was unwrapping the error but changed my mind and never took out the try/catch. I'll make a note to pull them out at some point.

Reply:
I just went ahead and removed them in my dep updates PR here: https://github.com/ghost-inspector/node-ghost-inspector/pull/27

Reply:
🙏🏻